### PR TITLE
Add circular queue (Basic Operations like Insert, Delete, Display elements etc)

### PR DESCRIPTION
**Pull request by shahina-bano**
I have added the solution code in python for issue #58. Kindly review it @plazzy99  @OjusWiZard

### CODE REVIEW

**Comment by Bushra109** [comment]
I am a GSSOC 21 participant.
I wish to work on this program in C++.
Kindly assign me this issue.
@plazzy99 @OjusWiZard

**Comment by shahina-bano** [comment]
Have a look @plazzy99 @Rohan27s

File: Code/Python/circular_queue.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+"""
+Circular Queue
+Problem Statement: Circular Queue is a linear data structure which follows FIFO principle
+                   The difference is that the last position is connected back to the first position to make a circle.
```
Comment (Rohan27s) [inline Code/Python/circular_queue.py:4]: Remove this space

File: Code/Python/circular_queue.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        if(self.quehead == -1):
+            print ("Underflow")
+
+        #else printing the elements of queue
+        else:
+            print("Elements in Queue: ")
+            for k in range(self.quehead, self.quetail + 1):
+                print(self.queue[k], end = " ")
+                
+
```
Comment (Rohan27s) [inline Code/Python/circular_queue.py:33]: Decrease double line space to single line spaces

File: Code/Python/circular_queue.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            print(self.queue[self.quehead])
+            self.quehead = self.quetail = -1
+
+        # deleting for the rest of condition
+        else:
+            print(self.queue[self.quehead])
+            self.quehead += 1
+
+
+# input the size of the queue
```
Comment (Rohan27s) [inline Code/Python/circular_queue.py:66]: Add print statement asking the user for the respective values

**Review by Rohan27s — CHANGES_REQUESTED** [review]
Code is fine just make the requested changes

File: Code/Python/circular_queue.py (hunk 2 above)
Comment (shahina-bano) [reply Code/Python/circular_queue.py:33]: @Rohan27s isn't there a rule to add 2 line spaces between functions.. for differentiating them in python.

**Review by Rohan27s — APPROVED** [review]
Thanks for your contribution!

**Comment by shahina-bano** [comment]
thank you @Rohan27s

**Pull request merged**